config.json

```
@@ -1582,6 +1582,18 @@
         ],
         "difficulty": 3
       },
+      {
+        "slug": "pop-count",
+        "name": "Pop Count",
+        "uuid": "8a9d6e1e-9188-44c9-8681-ade84d340dd0",
+        "practices": [
+          "bit-manipulation"
+        ],
+        "prerequisites": [
+          "bit-manipulation"
```

Alright, I think this is very easy without tail tail recursion, so those are the prerequisites that I think it needs:

```suggestion
          "bit-manipulation",
          "multiple-clause-functions",
          "pattern-matching",
          "recursion"
```